app/templates/components/public/event-map.hbs

```diff
@@ -1,17 +1,15 @@
 {{#if this.event.isMapShown}}
   <div class="eight wide column event-map">
     <h1>{{t 'Getting Here'}}</h1>
-    {{#if (eq this.mapConfig.display 'embed') }}
-      <iframe title="Map" class="g-map" src="https://maps.google.com/maps?q={{this.event.locationName}}&t=&z=15&ie=UTF8&iwloc=&output=embed" frameborder="0" scrolling="no"></iframe>
-    {{else}}
-      <GMap @markersFitMode="live" @lat={{37.744}} @lng={{-122.4367}} @address={{this.event.locationName}} @zoom={{2}} @class="google-maps" as |context|>
-        {{#g-map-address-marker context address=this.event.locationName as |markerContext|}}
-          {{#g-map-infowindow markerContext}}
-            {{this.event.locationName}}
-          {{/g-map-infowindow}}
-        {{/g-map-address-marker}}
-      </GMap>
-    {{/if}}
+    <LeafletMap @lat={{this.lat}} @lng={{this.lng}} @zoom={{this.zoom}} as |layers|>
+      <layers.tile @url="https://{s}.basemaps.cartocdn.com/light_all/{z}/{x}/{y}.png"/>
```

Comment:
Please use a free to use basemap, this is only free for non-commercial projects 

> Our basemaps can be used for free in applications and visualizations for non-commercial use. For commercial purposes, you will need an Enterprise license to ensure you have the best basemaps available.

Reply:
https://wiki.openstreetmap.org/wiki/Tiles#Servers

Reply:
op

Reply:
ok